DatadogSessionReplay/Tests/Mocks/UIKitMocks.swift

```diff
@@ -105,3 +105,34 @@ extension UITextContentType: RandomMockable {
 
     public static func mockRandom() -> UITextContentType { allCases.randomElement()! }
 }
+
+extension UIImage: RandomMockable {
+    /// Creates bitmap by randomising the value of each pixel.
+    public static func mockRandom() -> Self {
+        return mockRandom(width: .mockRandom(min: 10, max: 100), height: .mockRandom(min: 10, max: 100))
+    }
+
+    /// Creates bitmap of certain size by randomising the value of each pixel.
+    public static func mockRandom(width: Int, height: Int) -> Self {
```

Comment:
💡 Previously, some unit tests in SR used an actual [`.png` image](https://github.com/DataDog/dd-sdk-ios/tree/2.0.0-beta1/DatadogSessionReplay/Tests/Recorder/Utilties/Media.xcassets/dd_logo_v_rgb.imageset). To avoid re-adding `Media.xcassets` to the main `Datadog.xcworkspace`, I switched to generating random bitmap at runtime.